app/images.py

```diff
@@ -1,7 +1,10 @@
 import json
 import os
+import PIL
+from PIL import Image
 import random
 import requests
+import shutil
 import time
 import threading
```

Comment:
Reorganise imports according to [PEP8](https://www.python.org/dev/peps/pep-0008/#imports).